packages/frontend/CONTRIBUTING.md

```diff
@@ -27,31 +27,34 @@ yarn install
 
 ### Setting up envs
 
-A couple envs are required in order to start up the host frontend locally or
-build it for a release:
+A couple envs are needed in order to start up the host frontend locally or build
+it for a release:
 
 - `REACT_APP_INFURA_PROJECT_ID`: an infura project ID to be used for RPC access
   for various blockchains. You can get one [here](https://www.infura.io/).
 - `REACT_APP_WALLETCONNECT_PROJECT_ID`: a WalletConnect v2 project ID is
   required in order to enable WalletConnect based wallet connections. You can
-  get one by registering [here](https://cloud.walletconnect.com).
+  get one by registering [here](https://cloud.walletconnect.com). If you don't
+  care about WalletConnect support (for example in a local build, you can skip
+  defining this env).
 - `REACT_APP_FATHOM_SITE_ID`: id of an existing Fathom site, used to initialize
-  the anonymous tracking (this env is optional). You can get one by registering
+  the anonymous tracking (this env is optional if you're not building a
+  production bundle). You can get one by registering
   [here](https://app.usefathom.com).
 
 After you have these envs, you should create a `.env` file in the root of the
 package (or copy the `.env.example` available at the root of the package) and
 paste the values there.
 
-The `.env` file should also define 2 additional envs that will be used by the
-`build` command to generate code related to Fathom.
+The `.env` file must also define 2 additional envs that will be used by the
```

Comment:
We can specify that the `FATHOM` envs are not required if fathom won't be enabled.

Reply:
Yeah but the problem is they are required, otherwise the Fathom code won't be generated and even though it is not used the app will look for it, breaking.

Reply:
Ok, I'll update this section once the lib is updated, to actually make the envs optional, by generating "mocked" code.